extensions/script-libraries/semver/semver.livecodescript

```diff
@@ -0,0 +1,722 @@
+﻿script "semver"
```

Comment:
I think the stack should be called `com.livecode.script-library.semver` just to make it consistent with our other extensions.